backend/meson.build

```diff
@@ -5,10 +5,10 @@ backend_files = files(
 	'session/session.c',
 	'drm/backend.c',
 	'drm/drm.c',
-	'drm/drm-atomic.c',
-	'drm/drm-legacy.c',
-	'drm/drm-properties.c',
-	'drm/drm-util.c',
+	'drm/iface_atomic.c',
+	'drm/iface_legacy.c',
```

Comment:
I would just call these atomic.c and legacy.c

Reply:
I wanted it to be obvious that they were related to iface.h.

Reply:
I think that was already obvious.

Reply:
iface_atomic reads like hungarian notation imo